pinot-core/src/main/java/org/apache/pinot/core/data/manager/realtime/IngestionDelayTracker.java

```diff
@@ -76,6 +76,15 @@
 
 public class IngestionDelayTracker {
 
+  // Class to wrap supported timestamps collected for an ingested event
+  private static class IngestionTimestamps {
+    IngestionTimestamps(long ingestionTimesMs, long creationTimeMs) {
+      _ingestionTimeMs = ingestionTimesMs;
+      _creationTimeMs = creationTimeMs;
```

Comment:
`_ingestionTimeMs` is basically the time when a message was published to the last stream.
`_creationTimeMs` is the time when a message was published to the first stream.

Should we rename them to something like `lastStreamPublishTimeMs` and `firstStreamPublishTimeMs`? Or at least we need to have some comments to clearly specify what each variable is.

Reply:
Let me think of better names here...

Reply:
@sajjad-moradi / @jugomezv Is this `eventTime` which is when the event was first created or the `eventTime + delta` which is the time when the event entered the first upstream topic? 
`eventTime` is a common terminology. but not sure if that's what you mean by `_creationTimeMs` here

Reply:
I have followed the terminology suggested by @sajjad-moradi in commit 9bb0befe9a55c01bb3a5066a4b30137469819f38 but did not change the one for the earlier timestamp as that seems to have been around for a while

Reply:
@navina AFAIK in Kafka, depending on broker's configuration, timestamp on each event can be either  "LogAppendTime" or "CreateTime". Since that varies per Kafka installation, we can either choose a more generic name (nothing comes to my mind) or stick to the existing "stream ingestion/publish time".